pkgs/servers/dns/knot-resolver/default.nix

```diff
@@ -39,6 +50,13 @@ unwrapped = stdenv.mkDerivation rec {
     # ExecStart can't be overwritten in overrides.
     # We need that to use wrapped executable and correct config file.
     sed '/^ExecStart=/d' -i systemd/kresd@.service.in
+
+    # On x86_64-darwin loading by soname fails to find the libs, surprisingly.
+    # Even though they should already be loaded and they're in RPATH, too.
+    for f in daemon/lua/{kres,zonefile}.lua; do
+      substituteInPlace "$f" \
+        --replace "ffi.load("    "ffi.load('${lib.getLib knot-dns}/lib/' .. "
```

Comment:
```suggestion
        --replace "ffi.load(" "ffi.load('${lib.getLib knot-dns}/lib/' .. "
```

Reply:
I can go back there, as I wasn't sure.  The intention was to improve readability, as the line has lots of punctuation.

Reply:
OK, done.